src/SelfServe/PropertyDecorators.tsx

```diff
@@ -0,0 +1,108 @@
+import { DropdownItem, Info, InputType, UiType } from "../Explorer/Controls/SmartUi/SmartUiComponent";
+import { addPropertyToMap } from "./SelfServeUtils";
+
+interface Decorator {
+  name: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  value: any;
```

Comment:
```suggestion
  value: unknown;
```

Reply:
the compiler complains for this. I have a typecheck in the "addPropertyToMap()" function of SelfServeUtils which needs this to be any and not unknown.

Reply:
`any` is not needed. You can type this correctly: https://github.com/Azure/cosmos-explorer/commit/c857b9aab9b771d7c7fbcf18127a25e72ac5ec86

Reply:
got it, will make the change,